scripts/mpc_test.py

```diff
@@ -0,0 +1,1018 @@
+import glob
+import os
+import sys
+import time
+import subprocess
+import queue
+
+from setuptools import setup
+# from tkinter import _ImageSpec
+# from agents.navigation.controller import VehiclePIDController
+
+
+sys.path.append('../carla/dist/carla-0.9.9-py3.7-linux-x86_64.egg')
```

Comment:
Will have to change path based on system

Reply:
Yup. Will do that